api/src/main/java/gov/cms/ab2d/api/service/JobServiceImpl.java

```diff
@@ -36,4 +37,13 @@ public Job createJob(String resourceTypes, String url) {
 
         return jobRepository.save(job);
     }
+
+    public void cancelJob(String jobId) {
+        Job job = jobRepository.findByJobID(jobId);
```

Comment:
Once we have the other PR merged, we should extract job lookup and null check into a common private method, since "getJobByJobId" does the same thing